src/ui/castlekeysmouse.pas

```diff
@@ -781,7 +789,7 @@ function StrToKey(const S: string; const DefaultKey: TKey): TKey;
 { Construct TInputPressRelease corresponding to given event.
   @groupBegin }
 function InputKey(const Position: TVector2;
-  const Key: TKey; const KeyCharacter: Char): TInputPressRelease;
+  const Key: TKey; const KeyString: string; const ModifiersDown: TModifierKeys): TInputPressRelease;
 function InputMouseButton(const Position: TVector2;
```

Comment:
Better make the parameter `ModifiersDown` have default value, like

```
function InputKey(const Position: TVector2;
  const Key: TKey; const KeyString: string; const ModifiersDown: TModifierKeys = []): TInputPressRelease;
```

This function is public, so it's better to keep backward compatibility here.

Reply:
Done.
I left explicit `[]` specifications in `TStateDialogInput.ButtonOKClick`, `TStateDialogInput.ButtonCancelClick` in `castledialogstates.pas`, maybe remove those too?

Reply:
On the second thought, I believe it's worth removing (in case `InputKey` changes into something different in future empty array is "compatible" with everything so it might get a wrong initializing value). Removed those.